xbmc/addons/AddonManager.cpp

```diff
@@ -1035,4 +1042,44 @@ void CAddonMgr::FindAddons(ADDON_INFO_LIST& addonmap, const std::string& path)
   }
 }
 
+const bool CAddonMgr::IsFromTrustedRepo(const AddonPtr& addon) const
+{
+  if (m_trustedAddonRepos.size() != 0)
```

Comment:
I don't think this if statement is needed. The for loop will never start if size is 0 so I think this is redundant.